src/functions/public/Import-Json.ps1

```diff
@@ -0,0 +1,90 @@
+function Import-Json {
+    <#
+        .SYNOPSIS
+        Imports JSON data from a file.
+
+        .DESCRIPTION
+        Reads JSON content from one or more files and converts it to PowerShell objects.
+        Supports pipeline input for processing multiple files.
+
+        .EXAMPLE
+        Import-Json -Path 'config.json'
+        
+        Imports JSON data from config.json file.
+
+        .EXAMPLE
+        Import-Json -Path 'data/*.json'
+        
+        Imports JSON data from all .json files in the data directory.
+
+        .EXAMPLE
+        'settings.json', 'users.json' | Import-Json
+        
+        Imports JSON data from multiple files via pipeline.
+
+        .EXAMPLE
+        Import-Json -Path 'complex.json' -Depth 50
+        
+        Imports JSON data with a custom maximum depth of 50 levels.
+
+        .LINK
+        https://psmodule.io/Json/Functions/Import-Json/
+    #>
+
+    [CmdletBinding()]
+    param (
+        # The path to the JSON file to import. Supports wildcards and multiple paths. Can be provided via pipeline.
+        [Parameter(Mandatory, ValueFromPipeline, ValueFromPipelineByPropertyName)]
+        [Alias('FullName')]
+        [string[]]$Path,
+
+        # The maximum depth to expand nested objects. Default is 100.
+        [Parameter()]
+        [int]$Depth = 100
+    )
```

Comment:
Lets have this use the common default of ConvertFrom-Json which is 1024. Think we want to just follow that as a default, so have it optional without a value, and if we supply a value we use it.